presto-hive/src/test/java/com/facebook/presto/hive/TestHiveSplitSource.java

```diff
@@ -249,6 +253,69 @@ public void testEmptyBucket()
         assertEquals(getSplits(hiveSplitSource, OptionalInt.of(3), 10).size(), 0);
     }
 
+    @Test
+    public void testPreloadSplitsForGroupedExecution()
+            throws Exception
+    {
+        // TODO: Use Session::builder after HiveTestUtils::SESSION is refactored to Session.
```

Comment:
Remember to open an issue before we forgot it :)

Reply:
Created #12367